### PR TITLE
My Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by Jtaborda21**
### In this milestonte I:

- Added HTML and CSS.
- Added README and Linters.
- Added the toolbar.
- Added the headline section.
- Added responsive website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,11 +5,86 @@
...
+          </div>
+        </nav>
+      </header>
+      <section class="main" id="main">
+        <div class="main-container">
+          <h1>
+            Hey there. <span class="span-br"><br></span>
+            I’m Alex.
+          </h1>
+          <span class="h1-sub">I’m a software developer</span>
```
Comment (Qoosim) [inline index.html:40]: - [ ] The subtitle (if existing) uses a header tag hx (h2, ... h6) except `h1` which has been used for the main title. This is semantically correct and the screen readers and other technologies will thank you. Kindly change the subtitle to any of the above mentioned hx tags.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+
+I would like to thank to Microverse program.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (Qoosim) [inline README.md:200]: - [ ] You don't have this `LICENSE` file you're referencing to inside your file tree. As a result the link is broken at this moment. Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to properly set it up.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Jtaborda21, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
```
Comment (usorfaitheloho) [inline README.md:85]: [Optional] Kindly remove all commented out code from your README file as they are meant to give you a clearer insight on what to do and not to be part of your code 😄

**Review by usorfaitheloho — APPROVED** [review]
Hi @Jtaborda21  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**